tests/dialects/test_postgres.py

```diff
@@ -212,6 +212,15 @@ def test_postgres(self):
                 "trino": "SELECT APPROX_PERCENTILE(amount, 0.5)",
             },
         )
+        self.validate_all(
+            "SELECT PERCENTILE_CONT(0.5) WITHIN GROUP (ORDER BY amount)",
+            read={
+                "databricks": "SELECT PERCENTILE_APPROX(amount, 0.5)",
+                "presto": "SELECT APPROX_PERCENTILE(amount, 0.5)",
+                "spark": "SELECT PERCENTILE_APPROX(amount, 0.5)",
+                "trino": "SELECT APPROX_PERCENTILE(amount, 0.5)",
+            },
+        )
```

Comment:
Does this transpile correctly to DuckDB too? If not, is it easy to fix by e.g. including the `remove_within_group_for_percentiles` transformation for DuckDB?